packages/x-date-pickers/src/hooks/useLocaleText.ts

```diff
@@ -0,0 +1,5 @@
+import { PickerValidDate } from '../models';
+import { useLocalizationContext } from '../internals/hooks/useUtils';
+
+export const useLocaleText = <TDate extends PickerValidDate>() =>
```

Comment:
Are we sure about the name of the hook? 🤔 
Maybe `useLocaleTranslations` would make more sense? 🤷

Reply:
`useTranslations` maybe? :laughing:

Reply:
`next-intl` is already using this hook name: https://next-intl-docs.vercel.app/docs/usage/messages
It would probably be best to avoid a duplicate name on our end. 🤔 
Maybe just sticking to `localeText` isn't such a bad thing after all. 🙈 🤷

Reply:
`usePickersTranslations` then :angel: 
If we want to avoid duplication, I'd rather add "Pickers" to the most commonly used name

Reply:
I vote for `usePickersTranslations` then if you agree with it. 👌